dexbot/strategies/staggered_orders.py

```diff
@@ -876,7 +892,10 @@ def allocate_asset(self, asset, asset_balance):
                         or (self.mode == 'buy_slope' and asset == 'quote')
                         or (self.mode == 'sell_slope' and asset == 'base')
                     ):
+                        # TODO The amount may need to be calculated due to the 2nd opposite order used
                         opposite_asset_limit = opposite_order['base']['amount']
```

Comment:
`opposite_order['base']['amount']` of an order won't change when the order is partially filled? If yes, this PR does not fix the "too small new order" issue.

Reply:
The code still works, although the comment was wrong (and fixed).